docs/relnotes.rst

```diff
@@ -27,6 +27,47 @@ See https://github.com/spacetelescope/webbpsf/issues for currently open issues a
 Version History and Change Log
 -------------------------------
 
+Version 1.2.0
+=============
+We are pleased to announce the release of the latest version of WebbPSF version 1.2.0, now available on PyPi and GitHub. This release comes with new features and improvements including but not limited to:
+
+1. The addition of detector effects for JWST simulations. H2RG detector effects are included in two flavors, a simple ad hoc Gaussian convolution to model charge diffusion effects and a set of convolution kernels to model interpixel capacitance (IPC) and post-pixel coupling effects. We have found that these effects greatly improve the agreement between observations and simulations. See `JWST Detector Effects for more details. <https://webbpsf.readthedocs.io/en/latest/jwst_detector_effects.html>`_
+2. A new utility function for simulating matching PSFs to science data. See `Matching PSF sims to in-flight JWST data <https://webbpsf.readthedocs.io/en/latest/jwst_matching_psfs_data.html>`_. 
```

Comment:
In 2, the correct link is https://webbpsf.readthedocs.io/en/latest/jwst_matching_psfs_to_data.html (_to_ is missing)